src/Loader.php

```diff
@@ -246,7 +246,19 @@ protected function sanitiseVariableValue($name, $value)
                 /mx',
                 $quote
             );
+            $originalValue = $value;
             $value = preg_replace($regexPattern, '$1', $value);
+            if(($e = preg_last_error()) !== PREG_NO_ERROR) {
+                $l = get_defined_constants(true);
+                $l = $l['pcre'];
+                $l = array_flip($l);
+                throw new InvalidFileException($l[$e]);
+            }
+
+            // if they are identical, it failed to replace/match
+            if($originalValue === $value) {
+                throw new InvalidFileException("Invalid value");
```

Comment:
I don't think this exception message is very clear?

Reply:
I agree, I'll fix this.